README.md

```diff
@@ -36,7 +36,7 @@ $ bower install validator-js
 - **equals(str, comparison)** - check if the string matches the comparison.
 - **contains(str, seed)** - check if the string contains the seed.
 - **matches(str, pattern [, modifiers])** - check if string matches the pattern. Either `matches('foo', /foo/i)` or `matches('foo', 'foo', 'i')`.
-- **isEmail(str)** - check if the string is an email.
+- **isEmail(str [, allowDisplayName])** - check if the string is an email. `allowDisplayName` is a boolean which, if true, will also match `Display Name <email-address>`.
```

Comment:
You forgot to update this.

Also, minor nitpick, can you change `allowDisplayName` to `allow_display_name`? I know this isn't good JS but the rest of the validators accept options in this underscore style. I'd prefer to keep consistency for now and then revisit in v4 when I'll be making plenty of breaking changes, like changing options to camelcase.